src/HereAuth/HereAuth.php

```diff
@@ -221,7 +221,7 @@ public function onDisable(){
 	public function startUser(Player $player){
 		$this->database->loadFor($player->getName(), $player->getId());
 		if($player->spawned){
-			$player->sendMessage("[HereAuth] Your account data are being loaded. Please wait patiently; it shouldn't take long.");
+			$player->sendMessage("[HereAuth] Your account data is being loaded. Please wait patiently; it shouldn't take long.");
```

Comment:
You are referring to all the data as one thing when you use data, therefore using `is` would make more sense, unless you want to use datum, which in that case, an `are` would be adequate. I'll leave this up to you to decide on.